projects/storefrontlib/src/cms-components/content/link/link.component.ts

```diff
@@ -9,4 +9,14 @@ import { CmsComponentData } from '../../../cms-structure/page/model/cms-componen
 })
 export class LinkComponent {
   constructor(public component: CmsComponentData<CmsLinkComponent>) {}
+
+  /**
+   * Returns `_blank` to force opening the link in a new window whenever the
+   * `data.target` flag is set to rue.
+   */
+  getTarget(data: CmsLinkComponent): string | null {
+    return data.target === 'true' || ((data.target as any) as Boolean) === true
```

Comment:
we can drop as any or as Boolean, but not have both. Or, drop it both and have it similarly to `navigation.service`, where none of the above was provided

what do you think?

Reply:
much better